cmd/openshift-install/create.go

```diff
@@ -446,6 +459,88 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 	return nil
 }
 
+func waitForBaremetalBootstrapControlPlane(ctx context.Context, config *rest.Config) *clusterCreateError {
```

Comment:
Maybe move this logic into some file under `baremetal` so that we don't need to ping the core team every time we change something there?

Reply:
Done